### ISSUE TITLE
Better tags in the repository

### ISSUE THREAD

**Issue by Anders429** [Anders429/brood#102]
Up until now, I've been half-heartedly using the default tags provided by GitHub. But a lot of things that need attention, such as testing, continuous integration, and even specific features, don't exactly fall into the default categories. It would be better to have a custom set of tags for this repository, given the size of the code base (which is almost 20,000 LOC at this point).

**Comment by Anders429** [comment]
Here's a rough draft proposal:

# Categories
- Bug
- Crash
- Unsoundness
- Regression
- Performance
- Code Quality
- Dependencies
- Documentation
- Enhancement
- Testing
- Benchmarking
- Tracking Issue
- Continuous Integration
- Compilation

# Semantic Versioning
- Deprecation
- Patch
- Minor
- Major
- MSRV

Priority
- Low
- Medium
- High

Status
- Unstaffed
- Blocked on Dependency
- Blocked on Rust Feature
- Duplicate
- Needs Investigation
- User Error
- Won't Fix

Subject
- Storage
- Querying
- Systems
- Scheduling

Features
- serde
- rayon

**Comment by Anders429** [comment]
I've added nearly all of the tags enumerated above. I left out a few (such as some related to semantic versioning), figuring that if they are needed later they can be added.
